.github/workflows/olm_tests.yaml

```diff
@@ -0,0 +1,195 @@
+
+      - name: Set Go
+        uses: actions/setup-go@v3
+        with:
+          go-version: v1.18
+
+      - name: Set up gotestfmt
+        uses: gotesttools/gotestfmt-action@v2
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Setup and start KinD cluster
+        uses: ./.github/actions/kind
+
+      - name: Install OLM
+        run: |
+          kubectl create -f https://github.com/operator-framework/operator-lifecycle-manager/releases/download/${OLM_VERSION}/crds.yaml
+          # wait for a while to be sure CRDs are installed
+          sleep 1
+          kubectl create -f https://github.com/operator-framework/operator-lifecycle-manager/releases/download/${OLM_VERSION}/olm.yaml
+
+      - name: Create openshift-operator namespace and OperatorGroup
+        run: |
+          # Need to use openshift-operator namespace due to https://github.com/project-codeflare/codeflare-operator/issues/161
+          kubectl create namespace openshift-operators
+          kubectl create -f .github/resources-olm-upgrade/operatorgroup.yaml
+
+      - name: Deploy latest released CodeFlare operator from OLM
+        id: deploy
+        run: |
+          echo Deploying CodeFlare operator using Subscription
+          envsubst < .github/resources-olm-upgrade/catalogsource.yaml > ${{ env.TEMP_DIR }}/catalogsource.yaml
+          envsubst < .github/resources-olm-upgrade/subscription.yaml > ${{ env.TEMP_DIR }}/subscription.yaml
+
+          kubectl create -f ${{ env.TEMP_DIR }}/catalogsource.yaml
+          make wait-for-catalog-source
+
+          kubectl create -f ${{ env.TEMP_DIR }}/subscription.yaml
+
+          echo Waiting for Subscription to be ready
+          make wait-for-subscription
+
+          echo Waiting for Deployment to be ready
+          make wait-for-deployment -e TIMEOUT=60 -e DEPLOYMENT_NAME="codeflare-operator-manager" -e DEPLOYMENT_NAMESPACE="openshift-operators"
+        env:
+          CATALOG_SOURCE_NAME: "codeflare-olm-test"
+          CATALOG_SOURCE_NAMESPACE: "olm"
+          SUBSCRIPTION_NAME: "codeflare-operator"
+          SUBSCRIPTION_NAMESPACE: "openshift-operators"
+
+      - name: Store latest CSV version as PREVIOUS_VERSION env variable (used for bundle build)
+        run: |
+          CSV_VERSION=$(kubectl get ClusterServiceVersion -l operators.coreos.com/codeflare-operator.openshift-operators='' -n openshift-operators -o json | jq -r .items[].spec.version)
+          echo "PREVIOUS_VERSION=v$CSV_VERSION" >> $GITHUB_ENV
+
+      - name: Deploy CodeFlare stack (MCAD, KubeRay)
+        run: |
+          make setup-e2e
+
+      - name: Run e2e tests
+        run: |
+          export CODEFLARE_TEST_OUTPUT_DIR=${{ env.TEMP_DIR }}
+          echo "CODEFLARE_TEST_OUTPUT_DIR=${CODEFLARE_TEST_OUTPUT_DIR}" >> $GITHUB_ENV
+
+          set -euo pipefail
+          go test -timeout 30m -v ./test/e2e -json 2>&1 | tee ${CODEFLARE_TEST_OUTPUT_DIR}/gotest-original.log | gotestfmt
```

Comment:
I don't think we need to test the already published version of the operator. It **should** already be tested and verified by that point. It might even fail if a new e2e test is created which relies on new features.

Reply:
I suppose so, will remove the check

Reply:
Removed